internal/ignition/ignition.go

```diff
@@ -383,6 +380,30 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 	return nil
 }
 
+func (g *installerGenerator) bootstrapInPlaceIgnitionsCreate(installerPath string, envVars []string) error {
+	err := g.runCreateCommand(installerPath, "single-node-ignition-config", envVars)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create single node ignitions")
+	}
+
+	// In case of single node rename bootstrap Ignition file
+	err = os.Rename(filepath.Join(g.workDir, "bootstrap-in-place-for-live-iso.ign"), filepath.Join(g.workDir, "bootstrap.ign"))
+	if err != nil {
+		return errors.Wrapf(err, "Failed to rename bootstrap-in-place-for-live-iso.ign")
+	}
+
+	config := config_31_types.Config{Ignition: config_31_types.Ignition{Version: config_31_types.MaxVersion.String()}}
+
+	for _, file := range []string{masterIgn, workerIgn} {
```

Comment:
No need for the worker 
No need for the loop

Reply:
need worker. Without worker code will fail. Assisted expects it.